bindings/core/src/method/utils.rs

```diff
@@ -127,6 +127,12 @@ pub enum UtilsMethod {
         serial_number: u32,
         token_scheme_type: u8,
     },
+    /// Computes the hash of the given protocol parameters.
+    #[serde(rename_all = "camelCase")]
+    ProtocolParametersHash {
+        /// Network Protocol Parameters
```

Comment:
Tbh I wouldn't add such comments, I know there are already some but I'll probably clean it us and remove at some point. This is not a public crate, we don't care.